SPECS/python3-geomet/python3-geomet.spec

```
@@ -0,0 +1,53 @@
+Name:           python3-geomet
+Version:        0.2.1
+Release:        1%{?dist}
+Summary:        GeoJSON <-> WKT/WKB conversion utilities
+License:        ASL 2.0
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Group:          Development/Languages/Python
+Url:            https://github.com/geomet
+Source0:        https://github.com/geomet/geomet/archive/refs/tags/%{version}.tar.gz#/geomet-%{version}.tar.gz
+BuildRequires:  python3
+BuildRequires:  python3-libs
```

All python3 packages should require `python3-devel`. Since `python3-devel` pulls in `python3` and `python3-libs`, you can do the following:

```suggestion
BuildRequires:  python3-devel
```

---

Done.